docs/core/porting/project-structure.md

```diff
@@ -18,12 +18,12 @@ Learn how to create a solution that compiles for both .NET Framework and .NET Co
   - Allows you to write code for a specific .NET Framework version in your libraries through the use of compiler directives.
 
   *Unsupported scenarios:*
-  - Requires developers to use Visual Studio 2017 to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
+  - Requires developers to use Visual Studio 2017 os later versions to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
```

Comment:
```suggestion
  - Requires developers to use Visual Studio 2017 or a later version to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
```

Reply:
That's a good point. I'll open a PR later to fix all the occurrences of that in our docs then:
https://github.com/dotnet/docs/search?q=%22or+later+versions%22&unscoped_q=%22or+later+versions%22